sdk/python/feast/cli.py

```diff
@@ -870,5 +873,273 @@ def validate(
     exit(1)
 
 
+@cli.group(name="feast-permissions")
+def feast_permissions_cmd():
+    """
+    Access permissions
+    """
+    pass
+
+
+@feast_permissions_cmd.command(name="list")
+@click.option(
+    "--verbose",
+    "-v",
+    is_flag=True,
+    help="Print the resources matching each configured permission",
+)
+@tagsOption
+@click.pass_context
+def feast_permissions_command(ctx: click.Context, verbose: bool, tags: list[str]):
+    from bigtree import Node
+    from tabulate import tabulate
+
+    table: list[Any] = []
+    tags_filter = utils.tags_list_to_dict(tags)
+
+    store = create_feature_store(ctx)
+
+    permissions = store.list_permissions(tags=tags_filter)
+
+    root_node = Node("permissions")
+    roles: set[str] = set()
+
+    for p in permissions:
+        policy = p.policy
+        if not verbose:
+            _handle_not_verbose_permissions_command(p, policy, roles, table)
+        else:
+            if isinstance(policy, RoleBasedPolicy) and len(policy.get_roles()) > 0:
+                roles = set(policy.get_roles())
+                permission_node = Node(
+                    p.name + " " + str(list(roles)), parent=root_node
+                )
+            else:
+                permission_node = Node(p.name, parent=root_node)
+
+            for feast_type in p.types:
+                if feast_type in [
+                    FeatureView,
+                    OnDemandFeatureView,
+                    BatchFeatureView,
+                    StreamFeatureView,
+                ]:
+                    _handle_fv_verbose_permissions_command(
```

Comment:
can we move these methods separate file so that cli.py will act as a simple controller?